docs/installing-northstar/troubleshooting.md

```diff
@@ -90,68 +88,68 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
+    * _Restart game!_
 
-## The Main Menu is blank
+## The Main Menu is blank <a href="#blank-menu" id="blank-menu"></a>
 
-- Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.` / are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods) / included in the release zip).\
-- Try deleting `enabledmods.json` inside the R2Northstar folder as well.
-- Otherwise pay attention in console for your errors if you know what you're doing.
+* Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.` / are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods) / included in the release zip).\\
+* Try deleting `enabledmods.json` inside the R2Northstar folder as well.
+* Otherwise pay attention in console for your errors if you know what you're doing.
 
-## Adding Launch Options
+## Adding Launch Options <a href="#launch-opts" id="launch-opts"></a>
 
 Adding `-northstar` will start Northstar when launching from your launcher\
 Adding `-vanilla` will start the normal game when you have Northstar installed
 
-- For Steam `Your library -> Right click TF|2 -> Properties -> Launch Options -> -northstar or -vanilla`
-- For Origin `Your library -> Right click TF|2 -> Game Properties -> Advanced Launch Options -> Command Line Arguments -> northstar or vanilla`
+* For Steam `Your library -> Right click TF|2 -> Properties -> Launch Options -> -northstar or -vanilla`
+* For Origin `Your library -> Right click TF|2 -> Game Properties -> Advanced Launch Options -> Command Line Arguments -> northstar or vanilla`
 
 ## Access Violation
 
 If your error says `Access Violation | Attempted to read from 0x00000000` specifically DO NOT post just THAT.\
 The real error is most likely slightly higher. Please post that in issues or the discord
 
-## I can't play the Campaign
+## I can't play the Campaign <a href="#campaign" id="campaign"></a>
 
-Campaign is not supported in 1.4, but you could do one of these **"_fixes_"**
+Campaign is not supported in 1.4, but you could do one of these **"**_**fixes**_**"**
 
 1. Doing `ns_auth_allow_insecure 1` in the console
-    - [If you can't open the console](#i-cant-open-the-console)
+   * [If you can't open the console](troubleshooting.md#i-cant-open-the-console)
 2. Adding `-vanilla` to your launch options
-    - [How to add launch options](#adding-launch-options)
+   * [How to add launch options](troubleshooting.md#adding-launch-options)
 
-## Authentication Failed when clicking Launch Northstar
+## Authentication Failed when clicking Launch Northstar <a href="#lsx2" id="lsx2"></a>
 
-Before trying this check out [this section](#failed-to-initialize-origin-origin-core-seems-to-be-running-but-the-lsx-authentication-challenge-fai).\
+Before trying this check out [this section](troubleshooting.md#lsx-authentication-failed).\
```

Comment:
Also broken. On a sidenote, Gitbook doesn't seem to be a fan of `<a>` style refs :I